airflow-core/src/airflow/utils/module_loading.py

```diff
@@ -18,13 +18,34 @@
 from __future__ import annotations
 
 import pkgutil
+import re
 from importlib import import_module
 from typing import TYPE_CHECKING, Callable
 
 if TYPE_CHECKING:
     from types import ModuleType
 
 
+def is_valid_dotpath(path: str) -> bool:
```

Comment:
Not necessary for this PR but if you know of another piece of code in Airflow that could use this function then maybe at some point we should move this to a utils file or something